### DISCUSSION TITLE
About currency_exchange_rates table

### DISCUSSION THREAD

**Discussion by jimmymasaru** [firefly-iii/firefly-iii#5291] [Firefly III] [answered]
Hi,
I've checked the docs and reddit and it looks like the exchange rates functions have been removed but I can still see this table `currency_exchange_rates` in the database. Is it still used if I manually insert a row?

**Comment by JC5** [comment, marked as answer]
Nope.

**Reply by jimmymasaru** [reply]
In the dashboard, on the Y-axis, it puts my default currency but the chart contains accounts in different currencies, so I think if that's the case the Y-axis shouldn't show the currency symbol because it's a bit misleading as if it can convert currencies.

**Comment by z0rg0n** [comment]
I'm having the same problem.

I have three currencies enabled and the dashboard shows everything in the default currency, USD.

So I have say ¥30,000 but it shows on my dashboard as $30,000


![image](https://github.com/user-attachments/assets/4b026335-3a0b-4047-8e42-9c1a25b203d2)
